CMSgov/beneficiary-fhir-data
# [BFD-1256] Updated prod and test server certs

**JIRA Ticket:**
[BFD-1256](https://jira.cms.gov/browse/BFD-1256)

**User Story or Bug Summary:**
BFD prod and test app server certificates have either expired or are about to expire:

Prod
`
        Validity
            Not After : Sep 30 23:12:15 2021 GMT
`

Test
`
        Validity
           Not After : Sep 17 02:02:40 2021 GMT
`

### What Does This PR Do?

This ticket creates new prod and test certificate(s):

- prod-bluebutton-appserver-keystore.jks
- prod-bluebutton-appserver-public-cert.pem
- test-bluebutton-appserver-keystore.jks
- test-bluebutton-appserver-public-cert.pem

#### Prod
        Issuer: CN=bfd.cms.gov
        Validity
            Not Before: Sep 23 19:40:27 2021 GMT
            Not After : Sep 23 19:40:27 2023 GMT
        Subject: CN=bfd.cms.gov

#### Test
        Issuer: CN=test.bfd.cms.gov
        Validity
            Not Before: Sep 23 19:40:29 2021 GMT
            Not After : Sep 23 19:40:29 2023 GMT
        Subject: CN=test.bfd.cms.gov
---

If you're reviewing this PR, please check for these things in particular:
* Verify prod and test .PEM (public certs) are valid
* Ensure private keystore files (.jks) are encrypted (ansible-vault) and can be correctly decrypted when supplying the keybase password.

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then **you must** supply a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence here: **N/A**

    * Does this PR add any new software dependencies? 
    * Does this PR modify or invalidate any of our security controls?
    * Does this PR store or transmit data that was not stored or transmitted before?

* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR **should not be merged** unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons?

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following prerequisite changes have been fully deployed:

* N/A


### Submitter Checklist

I have gone through and verified that...:

* [x] I have named this PR and branch so they are [automatically linked](https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html) to the (most) relevant Jira issue. Ie: `BFD-123: Adds foo`
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [ x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. DC GEO, BB2, etc.).
    * Please review the standards every few months to ensure you're familiar with them.
